geo/src/algorithm/convex_hull/graham.rs

```diff
@@ -54,6 +54,10 @@ where
     for pt in points.iter() {
         while output.len() > 1 {
             let len = output.len();
+            if output[len - 1] == *pt {
```

Comment:
I'll be honest, I don't have a use case for `include_on_hull`, but based on the documentation it seems reasonable to expect that all the points in the input which are on the hull will be in the output - including **repeated** points.

Still, this change seems an improvement over the currently obviously broken behavior of returning a non-convex hull. 

Could you please document in the method signature and in the change log that repeated points will be removed?